pyccl/pk2d.py

```diff
@@ -527,3 +527,16 @@ def parse_pk2d(cosmo, p_of_k_a, is_linear=False):
             pk = cosmo.get_nonlin_power(name)
         psp = pk.psp
     return psp
+
+
+def parse_pk(cosmo, p_of_k_a=None):
+    """Helper to retrieve the power spectrum in the halo model."""
```

Comment:
This doesn't seem specific of the halo model. Document this a bit better.

Reply:
Can we leave this for later until we focus on the docstrings? I have been lazy writing docstrings for the new funcs I've been implementing because I know we'll go through them another time.

Reply:
please resolve all docstring-related, I'm taking care of them in 1053.